### ISSUE TITLE
[Chrome] DragOut fires when mouse goes over an element inside the drop area

### ISSUE THREAD

**Issue by marlon-tucker** [jkuri/ngx-uploader#340]
Hello,

On chrome, the drag out event fires when the user goes over elements which exist inside the drop area, therefore causes horrible flicker depending on what the drag over styles are.

It can be replicated on the demo page using Chrome Version 61.0.3163.79 (whatever the latest is).

I reckon confirming the event target is the same element as where the directive is declared will resolve this problem - but that's just a guess.

Thanks

Chrome

**Comment by webcat12345** [comment]
Same issue here. any clue about this issue?

**Comment by marlon-tucker** [comment]
I think I disabled pointer events via css and that sorted it

**Comment by webcat12345** [comment]
Yeah. it works fine, but I can't give a `pointer-events: none` to all child elements, that's why some of them are links and some of them are buttons and so on. Any better solution for this?

**Comment by webcat12345** [comment]
That could be an option. Thank you very much for your help
